.travis.yml

```diff
@@ -2,24 +2,36 @@ language: scala
 dist: trusty
 sudo: required
 group: edge
+
 jdk:
   - oraclejdk8
+  - oraclejdk9
+
 env:
   global:
   - WHITESOURCE_PROJECT_NAME: lagom-master
   - WHITESOURCE_PROJECT_TOKEN: b47efd29-4a67-4ece-94d5-a7f38c362d8a
   - secure: "qr/+kdqj2bdXRlsDR+akjsKTq8WJe0bBX+TKWaYCGcaGrqarOdCZcLteRxrE/uXmYD3rWJKhXkasUmUwEF45nTNX7X780nllFVA/AKUk3REDkLCpDkiwryMwdu/owx1xlK8dGyzPwrQhVulYotaK2tzv5r1bttemqBWcBOAgY7+HrnVhFNwS9tSrWp9CuDZotGANGd9JZBX+nITY2Ud59DcK5UFhfB4FJlqnIeCe201hD3Z9e6yWevPmu1vvV5qGlJ7h46jQvpXdvcNIlDDZ8e0oFqULjaUx6CPGUX5h9dQ9ZHvQgJJb2ALpTfR/Dxv/iM2dx3imGdjzklvJ8jcHX44jOQNucEwOxQk7dCGs+ie0L4KXrBqRxu0OwSC9U87CtliaC25Y35isfvOfNRgzL8mvzzZ+jSTy5s9wqBQZa9C04WIml4Azq8vL5ur26F1nxyZgHJBdQFJIBtR5m4av3oQLFEjCWwsVwYLR3JOP3wzrliT4BzNpeqppXzEJR2jt7MNgNmHH4am6suB8VS+7oxkn0R3GyRqMVEwqivj3ciisYBZZYGVn/SxmdSS3IhVgUivcLaCOA3gUh6Ww7bdTik7zB2AfCFYK5Lis0+STGXt9xaCf7Tp2+RxAGVFBP5K4QeFzzREG0aIlEgF5v2pn7wewve/p59NucPdxQE4c7x0="
   matrix:
-    - SCRIPT=test
+    - SCRIPT=test-2.11
+    - SCRIPT=test-2.12
     - SCRIPT=test-sbt
     - SCRIPT=test-maven
     - SCRIPT=test-documentation
     - SCRIPT=test-code-style
+
 # important to use eval, otherwise "&&" is passed as an argument to sbt rather than being processed by bash
 script: bin/$SCRIPT
 before_install:
     # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
     - rm ~/.m2/settings.xml
+    - if [[ "$(java -version 2>&1 | awk -F '"' '/version/ {print $2}')" > "1.9" ]]; then
+        sudo apt-get update -qq;
+        sudo /bin/echo -e oracle-java9-installer shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections;
+        sudo apt-get -o Dpkg::Options::="--force-confnew" install -y oracle-java9-installer oracle-java9-set-default oracle-java9-unlimited-jce-policy;
+        sudo update-java-alternatives -s java-9-oracle;
+      fi
```

Comment:
Is this still required? It looks like the Trusty images now include Oracle JDK 9: https://docs.travis-ci.com/user/reference/trusty/#JVM-(Clojure%2C-Groovy%2C-Java%2C-Scala)-images